sync-github/src/main/kotlin/gropius/sync/github/GithubDataService.kt

```diff
@@ -63,6 +71,7 @@ class GithubDataService(
 
     companion object {
         const val FALLBACK_USER_NAME = "github"
+        val TOKEN_WOUND_UP_IN_JAIL = ConcurrentHashMap<String, OffsetDateTime>()
```

Comment:
long term I still think this should be in the db

Reply:
i agree